app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java

```diff
@@ -147,7 +147,7 @@ public void onCreate(Bundle savedInstanceState) {
             loginCredentials.setVisibility(View.GONE);
         }
     }
-
+    // Hides the keyboard if the user's focus is not on the password (hasFocus is false).
```

Comment:
Would you mind using Javadoc-style comments? /** */ Thanks!

Reply:
Done!